src/core/load-fragment.c

```
@@ -6578,3 +6580,40 @@ int config_parse_tty_size(
 
         return config_parse_unsigned(unit, filename, line, section, section_line, lvalue, ltype, rvalue, data, userdata);
 }
+
+int config_parse_log_filter_patterns(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        ExecContext *c = ASSERT_PTR(data);
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        r = exec_parse_log_filter_pattern(c, rvalue);
+        if (ERRNO_IS_NOT_SUPPORTED(r)) {
+                log_syntax(unit, LOG_WARNING, filename, line, r,
+                           "PCRE2 support is not available, ignoring: %s=%s", lvalue, rvalue);
+                r = 0;
+        } else if (r == -EINVAL) {
+                log_syntax(unit, LOG_WARNING, filename, line, 0,
+                           "Regex pattern invalid, ignoring: %s=%s", lvalue, rvalue);
+                r = 0;
+        } else if (r == -ENOMEM) {
+                log_syntax(unit, LOG_WARNING, filename, line, r,
+                           "Failed to store log filtering pattern, ignoring: %s=%s", lvalue, rvalue);
+                r = 0;
+        }
+
+        return r;
```

Please do not silently ignore other errors:
```
if (r < 0) {
        log_syntax(...);
        return 0;
}

return 0;
```